dist/src/main/config/log4j2.xml

```diff
@@ -36,22 +33,19 @@
   </Appenders>
 
   <Loggers>
-    <Logger name="io.camunda.zeebe" level="${env:ZEEBE_LOG_LEVEL:-info}"/>
-    <Logger name="io.atomix" level="${env:ATOMIX_LOG_LEVEL:-info}"/>
-    <Logger name="io.camunda.operate" level="info" />
-    <Logger name="io.camunda.optimize" level="${env:OPTIMIZE_LOG_LEVEL:-info}" />
-    <Logger name="io.camunda.tasklist" level="info" />
-    <Logger name="org.elasticsearch" level="${env:ES_LOG_LEVEL:-warn}" />
-
-    <Root level="info">
+    <Logger name="io.camunda" level="${env:CAMUNDA_LOG_LEVEL:-INFO}" />
+    <Logger name="io.camunda.zeebe" level="${env:ZEEBE_LOG_LEVEL:-${env:CAMUNDA_LOG_LEVEL-INFO}}" />
+    <Logger name="io.atomix" level="${env:ATOMIX_LOG_LEVEL:-${env:CAMUNDA_LOG_LEVEL-INFO}}" />
+    <Logger name="io.camunda.optimize" level="${env:OPTIMIZE_LOG_LEVEL:-${env:CAMUNDA_LOG_LEVEL-INFO}}" />
+    <Logger name="org.elasticsearch" level="${env:ES_LOG_LEVEL:-WARN}" />
+    <Logger name="org.springframework" level="INFO" />
+
+    <Root level="WARN">
```

Comment:
:+1: I like this change. In the PR description you call this a "fix" but best I can tell it was actually intended to be info before?

Reply:
You're very right :) I got bamboozled by my own changes, where I initially changed it to something else first, and then back :see_no_evil: